### ISSUE TITLE
knownhosts updating should upsert instead of append

### ISSUE THREAD

**Issue by wraithm** [bitnomial/aws-ec2-knownhosts#4]
Currently, the knownhosts updating command just appends keys to the known hosts file creating duplicates if the host already existed in the knownhosts file. Instead, this tool should upsert, ie. append if the host doesn't exist and update (or delete the old entry and insert the new one) the key if it does.

**Comment by wraithm** [comment]
```
        jq -r 'map(.fqdn) | .[]' ./aws-ec2-pubkeys.json \
            | xargs -I{} ssh-keygen -R {}
```

**Comment by wraithm** [comment]
Solved by #19
